packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -156,7 +156,7 @@ const selectPatterns = createSelector(
 				categoryId,
 				hasCategory: ( item, currentCategory ) => {
 					if ( item.type === PATTERN_TYPES.user ) {
-						return item.wp_pattern_category.some(
+						return item.wp_pattern_category?.some(
```

Comment:
There's another `item.wp_pattern_category.some( ... )` call 16 lines below. I assume you intentionally left it untouched because it's guarded with `! item.wp_pattern_category?.length ||`, but I would feel safer if we added the `?` there too — just in case someone in the future refactors that code and in the process forgets to guard it.

Reply:
Yes, that was why.
I updated this location and found one more that skipped earlier.